README.md

```diff
@@ -10,11 +10,20 @@ npx react-native init MyProject --template https://github.com/brains-and-beards/
 
 Project inits with some demo usage files. You can browse them to see the proposed usage patterns. After init feel free to remove them:
 
+- `src/__mocks__/fixtures`
```

Comment:
one readme you see once browsing template's repo, the other one gets copied into your project. It might be redundant and not, I didn't split it this way so I'm leaving both for now